sumy/utils.py

```diff
@@ -49,15 +50,20 @@ def expand_resource_path(path):
 
 
 def get_stop_words(language):
-    path = expand_resource_path("stopwords/%s.txt" % language)
-    if not exists(path):
+    try:
+        stopwords_data = pkgutil.get_data("sumy", "data/stopwords/%s.txt" % language)
```

Comment:
Yes, I just wanted to have resources at the single place. But I see function `expand_resource_path` is used only at one place in this project, so I think it's OK. Sorry